### ISSUE TITLE
Default compiler label broken

### ISSUE THREAD

**Issue by duggelz** [google/subpar#71]
Attempting to build a par file gives this:

```
$ bazel build //...
ERROR: /path/BUILD:3:1: no such package 'compiler': BUILD file not found on package path and referenced by '//:foo.par'
ERROR: Analysis of target '//:foo.par' failed; build aborted: no such package 'compiler': BUILD file not found on package path
INFO: Elapsed time: 0.271s
FAILED: Build did NOT complete successfully (0 packages loaded)
```

Cause: the default value of the `compiler` attribute is being evaluated in the wrong context.  See #70

**Comment by fahhem** [comment]
This is closed now that #70 is merged, right?

**Comment by duggelz** [comment]
Yes, this is now available, tagged as the 1.2.0 release.
